rest/route/version.go

```diff
@@ -67,6 +73,7 @@ func (vh *versionHandler) Run(ctx context.Context) gimlet.Responder {
 // PATCH /rest/v2/versions/{version_id}
 
 type versionPatchHandler struct {
+	// Will activate the version if true and deactivate if false.
```

Comment:
This is an example of a parameter that used to be marked as required. Is there a way to have query parameters have the same red "required" syntax or no since this is just parsing regular comments?
<img width="552" alt="image" src="https://github.com/evergreen-ci/evergreen/assets/26798134/55cdff5e-efb4-44fe-9d8e-edfb6de3c537">

Reply:
Might be worth a follow up ticket to add these flags.